### PR TITLE
Portfolio's accessibility

### PR DESCRIPTION
**Pull request by GloireIrenge**
:world_map:  **Changes made for accessibility**

We link the line of code for each change we made to make our web page accessible. 
Page titles → We changed the title to title that is well described to users
https://github.com/GloireIrenge/My_Portfolio/commit/51ab90f21d2dcda65a25ce6dc6439d6021104e18#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R8

Image text alternatives :desert_island: → We add the alt attribute for each image.
https://github.com/GloireIrenge/My_Portfolio/commit/56ba9cfd24a7bc373a54781ee7015139c1767a38#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L287
https://github.com/GloireIrenge/My_Portfolio/commit/56ba9cfd24a7bc373a54781ee7015139c1767a38#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R287
https://github.com/GloireIrenge/My_Portfolio/commit/56ba9cfd24a7bc373a54781ee7015139c1767a38#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R284
https://github.com/GloireIrenge/My_Portfolio/commit/56ba9cfd24a7bc373a54781ee7015139c1767a38#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R285

Text headings → no issues.

Color contrast → We adjusted the colors to have better contrast.
https://github.com/GloireIrenge/My_Portfolio/commit/db599d01b0bf02b71067ece0a4aee46a05ade172#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4R412
https://github.com/GloireIrenge/My_Portfolio/commit/db599d01b0bf02b71067ece0a4aee46a05ade172#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4R407

Resize → We add the font size of the placeholder 
https://github.com/GloireIrenge/My_Portfolio/commit/fda19796e3e2d2c80c2ba667ad3d322ad185f3a8#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4R292

Interaction → We add some focus interaction changes
https://github.com/GloireIrenge/My_Portfolio/commit/06cf240058af6671852b5e4185457d5193b3124c#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4R672
https://github.com/GloireIrenge/My_Portfolio/commit/06cf240058af6671852b5e4185457d5193b3124c#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4R666
https://github.com/GloireIrenge/My_Portfolio/commit/06cf240058af6671852b5e4185457d5193b3124c#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4R660

Multimedia → no issues found.

The basic structure of the page → no issues found.

color-constract → We change the paragraph color to let it be shown correctly
https://github.com/GloireIrenge/My_Portfolio/commit/06cf240058af6671852b5e4185457d5193b3124c#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4R736
Headings →We change h1 headings for HTML best practice
https://github.com/GloireIrenge/My_Portfolio/commit/7f819817e5db987b59dbf2d67de401bc607d79f1#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R70
https://github.com/GloireIrenge/My_Portfolio/commit/7f819817e5db987b59dbf2d67de401bc607d79f1#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R209
Accessibility on devices  → We fix the size so that text will be shown without scrolling horizontally
https://github.com/GloireIrenge/My_Portfolio/commit/f7b8bf6136239c29047a315cd528d1fee59d549f#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4R856
https://github.com/GloireIrenge/My_Portfolio/commit/f7b8bf6136239c29047a315cd528d1fee59d549f#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4R873
https://github.com/GloireIrenge/My_Portfolio/commit/f7b8bf6136239c29047a315cd528d1fee59d549f#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4R934
Resize the placeholder → We add the placeholder font size so that users can easily see it but if it does not show on the deployed page , maybe it will display correctly after merging.
https://github.com/GloireIrenge/My_Portfolio/commit/6f5b6987041c102e8b3f19f6371050fddc0c339e#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4R672

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,11 +35,11 @@
...
-                                        <li><a href="#"><img class="icon" alt="social_icon" src="imgs/in.svg"></a> </li>
-                                        <li><a href="#"><img class="icon" alt="social_icon" src="imgs/Enabled.svg"></a></li>
-                                        <li><a href="#"><img class="icon" alt="social_icon" src="imgs/Vector.svg"></a></li>
-                                        <li><a href="#"><img class="icon" alt="social_icon" src="imgs/Disabled.svg"></a></li>
+                                        <li><a href="#"><img class="icon" alt="Github icon" src="imgs/gi.svg"></a></li>
+                                        <li><a href="#"><img class="icon" alt="Linkedin Icon" src="imgs/in.svg"></a> </li>
+                                        <li><a href="#"><img class="icon" alt="Social media Icon" src="imgs/Enabled.svg"></a></li>
+                                        <li><a href="#"><img class="icon" alt="social media icon" src="imgs/Vector.svg"></a></li>
+                                        <li><a href="#"><img class="icon" alt="Social media Icon" src="imgs/Disabled.svg"></a></li>
                                   </ul>
```
Comment (HAMMAS-SALEEM) [inline index.html]: - [ ] In line 70 you have another h1 tag which is completely against the [hierarchy of headings](https://www.w3.org/WAI/test-evaluate/preliminary/#headings). Also in many places, you didn't follow the [hierarchy of headings](https://www.w3.org/WAI/test-evaluate/preliminary/#headings). Please follow the hierarchy :+1:

Comment (HAMMAS-SALEEM) [inline index.html]: Your website should not give a horizontal scroll when it is resized or zoomed in up to 200% maximum. You can take this [portfolio](https://orozcoding.github.io/myportfolio/) as an example. You can zoom in on this [portfolio](https://orozcoding.github.io/myportfolio/) up to 200% and you won't find any horizontal scroll. Please fix it :+1:

File: index.html [hunk 2]
```diff
@@ -35,11 +35,11 @@
         <div class="body-class">
```
Comment (HAMMAS-SALEEM) [inline index.html]: ### Image 1
cards (copy)

### Image 2
color-blind (copy)

### Image 3
forms-of-focus (copy)

**Changes**
- [ ] You don't have a color contrast of 4.5:1 for these texts shown in the images. 
- [ ] Please add CSS :focus effect on each input item to make it visible for color-blind people focusing.
- [ ] Please add labels to each input item for accessibility purposes.
- [ ] Please make the placeholders a little bigger. Please fix them :+1:

File: index.html (hunk 1 above)
Comment (HAMMAS-SALEEM) [inline index.html]: # Requirements missing are in the red box :blush:

requirement-last (copy)

- [ ] It's good that you have added the summary and added some points of accessibility changes but you didn't follow the requirements completely. You need to add all the points which are also required in the requirements but you have added some points. You can take a look at this [pull request summary](https://github.com/orozCoding/myportfolio/pull/8) to get a good idea :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey Team (@GloireIrenge, @spydaspider) :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: Following Linters 
:heavy_check_mark: Following GitHub Flow


Good job on implementing some of the changes requested by the previous review but there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (victorokoroji) [reply index.html]: - [ ] As the previous reviewer suggested, Your website should not give a horizontal scroll when resized or zoomed in up to 200% maximum. The user doesn't have to scroll horizontally to be able to read the text. Please fix it 👍

File: index.html (hunk 2 above)
Comment (victorokoroji) [reply index.html]: - [ ] Kindly make your placeholder a little bigger and bolder as it is currently faint. This enables those with eye impairment to see clearly what is being written. Please ensure you adjust it as noted by the previous reviewer
- 
image

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @GloireIrenge , 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- Linters are working properly ✅ 


 ## Required Changes ♻️

- [ ] I can see that you have many  `h1` tags in your portfolio which is completely against the [hierarchy of headings](https://www.w3.org/WAI/test-evaluate/preliminary/#headings). Kindly implement these changes as suggested by the previous reviewer. 👍

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @GloireIrenge     , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**